src/runtime/type.ts

```diff
@@ -294,9 +296,9 @@ export class EntityType extends Type {
     // Spit MyTypeFOO to My Type FOO
     if (this.entitySchema.name) {
       return this.entitySchema.name.replace(/([^A-Z])([A-Z])/g, '$1 $2')
-                                   .replace(/([A-Z][^A-Z])/g, ' $1')
-                                   .replace(/[\s]+/g, ' ')
-                                   .trim();
+        .replace(/([A-Z][^A-Z])/g, ' $1')
```

Comment:
Undo these changes (since unrelated to the PR).

Reply:
Done!